### PR TITLE
Complete JavaScript methods exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
javascript-methods

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+console.log('value of firstBook', firstBook);
+
+var js = {
+    title: 'JavaScript for Impatient Programmers',
+    author: 'Dr. Axel Rauschmayer'
+  };
+var css = {
+    title: 'CSS Secrets',
+    author: 'Lea Verou'
+  };
```
Comment (shawnkost) [inline javascript-methods/main.js]: No `var`! This should be using `const`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple tweaks here Dennis!

File: javascript-methods/main.js (hunk 1 above)
Comment (dennis-pnguyen) [reply javascript-methods/main.js]: Screenshot 2023-07-12 103009
Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**